.github/workflows/pull.yml

```diff
@@ -24,6 +24,7 @@ jobs:
           - os: ubuntu-latest
             go: "1.19"
             test: true
+            testWithFips: true
```

Comment:
The main version of the compiler we're building at now is 1.18 - shouldn't this test be moved to 1.18 ubunutu-latest?

Also it would be good to perform the test with both fips-enabled and iwthout-fips code by cloning the matrix entry.

Reply:
actually it doesn't matter since the fips test run inside a docker container. I can move it to 1.18 though

Reply:
`Also it would be good to perform the test with both fips-enabled and iwthout-fips code by cloning the matrix entry.`

Could you let me know more about this @byo ? Because I think these two lines does that right

```
            test: true
            testWithFips: true
```

You'd want me to do it for 1.18 too?

Reply:
Ah, OK, didn't get that it does two separate steps now.

What I would suggest then is to actually add a separate built step that only has the `testWithFips` set without neither `test` nor `testWithMinio` set to true so that it is easier to understand.

Reply:
ack

Reply:
done